js/components/main.js

```diff
@@ -822,19 +822,17 @@ class Main extends ImmutableComponent {
   }
 
   get customTitlebar () {
-    const customTitlebarEnabled = isWindows
+    const customTitlebarEnabled = true // isWindows TODO revert before merge
     const captionButtonsVisible = customTitlebarEnabled
     const menubarVisible = customTitlebarEnabled && (!getSetting(settings.AUTO_HIDE_MENU) || this.props.windowState.getIn(['ui', 'menubar', 'isVisible']))
-    const selectedIndex = this.props.windowState.getIn(['ui', 'menubar', 'selectedIndex'])
+    const tempContextMenuIndex = this.props.windowState.getIn(['ui', 'contextMenu', 'selectedIndex'])
     return {
       enabled: customTitlebarEnabled,
       captionButtonsVisible: captionButtonsVisible,
       menubarVisible: menubarVisible,
       menubarTemplate: menubarVisible ? this.props.appState.getIn(['menu', 'template']) : null,
-      menubarSelectedIndex: selectedIndex,
-      contextMenuSelectedIndex: typeof selectedIndex === 'object' && Array.isArray(selectedIndex) && selectedIndex.length > 1
-        ? selectedIndex.slice(1)
-        : null,
+      menubarSelectedIndex: this.props.windowState.getIn(['ui', 'menubar', 'selectedIndex']),
+      contextMenuSelectedIndex: tempContextMenuIndex !== null ? tempContextMenuIndex : [0],
```

Comment:
I believe it would be useful to keep the previous logic here for the check (because the index should always be an array with length > 0).  The new default to `[0]` make perfect sense, because this is what we'd like for when autofill pops up :smile: